src/Microsoft.OData.Core/JsonLight/ODataJsonLightValueSerializer.cs

```diff
@@ -16,6 +16,7 @@ namespace Microsoft.OData.JsonLight
     using Microsoft.OData.Metadata;
     using Microsoft.OData.Edm;
     using ODataErrorStrings = Microsoft.OData.Strings;
+    using System.Threading.Tasks;

```

Comment:
sorting

Reply:
@xuzhg I'm not sure how I missed that. Fixed